test/e2e/e2e-sya/e2e.attendanceAllowance.en.test.js

```diff
@@ -11,6 +11,6 @@ const hasDwpIssuingOffice = testDataEn.benefitTypes[5].hasDwpIssuingOffice;
 
 Feature(`${language.toUpperCase()} - Attendance Allowance E2E SYA - Full Journey`);
 
-Scenario(`${language.toUpperCase()} - ${benefitCode} E2E SYA Journey @fullFunctional @e2e`, ({ I }) => {
+Scenario(`${language.toUpperCase()} - ${benefitCode} E2E SYA Journey @fullFunctional @e2e @fullFunctional-today`, ({ I }) => {
```

Comment:
Possibly -today is not the right tag name.

Reply:
good catch @pats-john , removed it.